pkgs/development/libraries/catboost/default.nix

```diff
@@ -0,0 +1,111 @@
+{ lib
+, config
+, stdenv
+, fetchFromGitHub
+, cmake
+, libiconv
+, llvmPackages
+, ninja
+, openssl
+, python3
+, ragel
+, yasm
+, zlib
+, cudaSupport ? config.cudaSupport or false
+, cudatoolkit
```

Comment:
This way of consuming cuda is obsolete: `cudatoolkit` is an alias to `cudaPackages.cudatoolkit`. Attributes outside `cudaPackages` consume the entire `cudaPackages` scope and then access its individual elements, such as `cuda_nvcc` or `cudatoolkit`: https://nixos.org/manual/nixpkgs/unstable/#cuda